packages/sitecore-jss-react/src/enhancers/usePersonalization.test.tsx

```diff
@@ -0,0 +1,204 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
According to the latest JSS best practices file names should be in lower case with dash separator.
E.g. use-personalization.test.tsx

Reply:
![image](https://user-images.githubusercontent.com/16399100/120666776-87a76d80-c495-11eb-9c38-9e2f35d43172.png)
It seems we are aligned with the existing naming